src/installing-workflow/configuring-object-storage.md

```diff
@@ -1,140 +1,116 @@
 # Configure Object Storage
 
-A variety of Deis components rely on an object storage system to do their work. These components are:
+A variety of Deis Workflow components rely on an object storage system to do their work including storing application slugs, Docker images and database logs.
 
-- [builder](https://github.com/deis/builder)
-- [slugbuilder](https://github.com/deis/slugbuilder)
-- [slugrunner](https://github.com/deis/slugrunner)
-- [controller](https://github.com/deis/controller)
-- [registry](https://github.com/deis/registry)
-- [database](https://github.com/deis/postgres)
+Deis Workflow ships with [Minio](http://minio.io) [component](https://github.com/deis/minio) by default, which provides in-cluster, ephemeral object storage. This means that _if the Minio server crashes, all data will be lost_. Therefore, **Minio should be used for development or testing only**.
 
-These components are flexible and can work out of the box with almost any system that is compatible with the [S3 API](http://docs.aws.amazon.com/AmazonS3/latest/API/APIRest.html).
+## Configuring off-cluster Object Storage
 
-## Minio
+Every component that relies on object storage uses two inputs for configuration:
 
-Additionally, Deis ships with a [Minio](http://minio.io) [component](https://github.com/deis/minio) by default, which provides in-cluster, ephemeral object storage. This means that _if the Minio server crashes, all data will be lost_. Therefore, **Minio should be used for development or testing only**.
+1. Component-specific environment variables (e.g. `BUILDER_STORAGE` and `REGISTRY_STORAGE`)
+2. Access credentials stored as a Kubernetes secret named `objectstorage-keyfile`
 
-In our beta release, the components listed above are configured by default to automatically use the Minio [service][k8s-service] for object storage.
+The helm chart for Deis Workflow can be easily configured to connect Workflow components to off-cluster object storage. Deis Workflow currently supports Google Compute Storage, Amazon S3 and Azure Blob Storage.
 
-# Configuring the Deis Components
+* **Step 1:** Create buckets for each of the workflow subsystems: builder, registry and database
+* **Step 2:** Create user credentials that have access to the storage buckets from step 1
```

Comment:
This was motivated by the encouraged separation of concerns. IAM creds for the writers, that may or may not have permission to manipulate buckets.

```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Action": [
                "s3:*"
            ],
            "Effect": "Allow",
            "Resource": [
                "arn:aws:s3:::bucket-registry",
                "arn:aws:s3:::bucket-registry/*",
                "arn:aws:s3:::bucket-database",
                "arn:aws:s3:::bucket-database/*",
                "arn:aws:s3:::bucket-builder",
                "arn:aws:s3:::bucket-builder/*"
            ]
        }
    ]
}
```